packages/app/src/static-props/get-data.ts

```diff
@@ -131,12 +129,15 @@ export function selectVrData<T extends keyof Regionaal = never>(
   return (context: GetStaticPropsContext) => {
     const vrData = getVrData(context);
 
-    const selectedVrData = {} as Pick<Regionaal, T>;
+    const selectedVrData: Record<string, any> = {};
```

Comment:
Ah bummer I didn't realize you then also had to type this still. You can probably use string, unknown :)

Reply:
You could avoid using a temporary variable by using reduce. This is using lodash set:

```suggestion
    const selectedVrData: Pick<Regionaal, T> = metrics.reduce((acc, p) => set(acc, p, vrData.data[p]), {})
```